bookstore/server.js

```diff
@@ -1,22 +1,18 @@
 const cds = require ('@sap/cds')
 
-// Add mashup logic
-cds.once('served', require('./srv/mashup'))
-
 // Add routes to UIs from imported packages
 cds.once('bootstrap',(app)=>{
-  app.serve ('/bookshop') .from ('@capire/bookshop','app/vue')
-  app.serve ('/reviews') .from ('@capire/reviews','app/vue')
-  app.serve ('/orders') .from('@capire/orders','app/orders')
-  app.serve ('/data') .from('@capire/data-viewer','app/viewer')
+  app.serve ('/admin-authors') .from ('@capire/fiori','app/admin-authors')
+  app.serve ('/admin-books') .from ('@capire/fiori','app/admin-books')
+  app.serve ('/browse-books') .from ('@capire/fiori','app/browse')
```

Comment:
apps are not loaded correctly, as they don't have an index.html file

Reply:
in ./fiori they are loaded through a `fiori-apps.html`
However using this (e.g. `app.serve ('/fiori-apps') .from ('@capire/fiori','app/fiori-apps.html')`)instead doesn't work either, because the `fioriSandboxConfig.json` is not found, hence it's showing a generic FLP with random apps missing the ones we would want to show